inception/inception-search-core/src/main/java/de/tudarmstadt/ukp/inception/search/StatisticsResult.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Licensed to the Technische Universität Darmstadt under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The Technische Universität Darmstadt
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package de.tudarmstadt.ukp.inception.search;
+
+import java.util.ArrayList;
+import java.util.Map;
+
+import de.tudarmstadt.ukp.clarin.webanno.model.Project;
+import de.tudarmstadt.ukp.clarin.webanno.security.model.User;
+
+public class StatisticsResult
+{
+
+    private Double upperDocSize;
+    private Double lowerDocSize;
+    private User user;
+    private Project project;
+
+    private Map<String, Map<String, Double>> allResults;
+    private Map<String, Map<String, Double>> nonTrivialResults;
+
+    public StatisticsResult(StatisticRequest aStatisticRequest,
+            Map<String, Map<String, Double>> allResults,
```

Comment:
Do we know which metrics we have? If yes, we could accessors here to get a statistic for a certain metric, e.g. getMean(String layerName, String featureName) or so.

Reply:
We do not know them apriori, but we can access them after a StatisticsResult object is created. I wrote a method getStatistic(String aMetric, String aLayerName, String aFeatureName) where the statistic is an input, too.

Reply:
I thought that we know what metrics the MTAS can give us, so we do not need a map but just can use a data object. Can you explain why we do not know the metrics ahead of time?

Reply:
Also see my comment on the next question.
Yes, we know what metrics MTAS can give us. 
I imagined it like this: the user ticks boxes in inception to choose which metrics should be calculated for her. These boxes translate to a regex which tells MTAS which statistics to calculate. 
I have no idea if runtime speed is an issue here at all (I only tested with 10-word corpora). If it is no issue, then we can always compute all statistics and only show those to the user which she actually wants to see. 
I think my current approach is more flexible but maybe this flexibility is unnecessary.

Reply:
Can you maybe test it on a large corpus? We have a large project generator somewhere 
@reckart Do you know where the script is we use to generate and test large projects?

Reply:
@UWinch did you use a generator script to build the "tiny" project?